src/layouts/index.js

```diff
@@ -64,9 +64,9 @@ const conferencesItems = [
     title: "eBPF Track (LPC)",
     subitems: [
       {
-        name: "2022 CFP",
+        name: "2022",
         linkUrl:
-          "https://lore.kernel.org/bpf/cd33ca74-aec9-ff57-97d5-55d8b908b0ba@iogearbox.net/",
+          "https://lpc.events/event/16/sessions/131/#20220912",
```

Comment:
```suggestion
          "https://lpc.events/event/16/sessions/131/#all",
```

This will display the schedule for the three days at once, instead of the first day only

Reply:
Ah never mind I see Daniel has updated this link in https://github.com/ebpf-io/ebpf.io-website/pull/223 already